src/client/services/date/js/js.date.service.ts

```diff
@@ -1,7 +1,10 @@
 import { DateService } from '../date.service';
 
 export class JsDateService implements DateService {
-	formatToHumanReadableDate(date: string): string {
+	formatToHumanReadableDate(date: string | Date): string {
+		if(date instanceof Date) {
+			date.toLocaleDateString('fr-FR', { dateStyle: 'long' });
+		}
 		return new Date(date).toLocaleDateString('fr-FR', { dateStyle: 'long' });
```

Comment:
```suggestion
	formatToHumanReadableDate(date: string | Date): string {
                const result = (date instanceof Date) ? date : new Date(date);
		return result.toLocaleDateString('fr-FR', { dateStyle: 'long' });
```
(pour éviter la duplication, et tout ce qui va avec)

Reply:
D'ailleurs il manque un return dans le if nan ? Je pense que si tu fais un `new Date(date)` avec `date instanceof Date`, ça doit juste faire une copie et donc marcher quand même 🤔

Reply:
oulala je l'avais pas vu ça !!!